pipeline/etl.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+from extract import main as emain
+from transform import main as tmain
+
+def main(topic: list[str]) -> pd.DataFrame:
```

Comment:
needs a module docstring please!

Reply:
```suggestion
def main(topic: list[str]) -> pd.DataFrame:
"""Main function to extract and transform the data from BlueSky and Google trends"
```

Reply:
This was just a test file to run extract and transform back to back. It will be created again once load has been finished